src/core/bpf-firewall.c

```diff
@@ -381,7 +382,7 @@ static int bpf_firewall_add_access_items(
                         key_ipv4->prefixlen = a->prefixlen;
                         memcpy(key_ipv4->data, &a->address, sizeof(uint32_t));
 
-                        r = bpf_map_update_element(ipv4_map_fd, key_ipv4, &value);
+                        r = bpf_map_update_elem(ipv4_map_fd, key_ipv4, &value, BPF_ANY);
```

Comment:
how does libbpf do error handling? we use negative `return -errno` style returns, and what does libbpf do? if they return -1 and set errno, this needs more love